firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -272,7 +272,7 @@ extension String {
                 key: "Addresses.BottomSheet.UseSavedAddressBottomSheet.v124",
                 tableName: "BottomSheet",
                 value: "Use a saved address?",
-                comment: "Title displayed on the bottom sheet for prompting users to use a saved address from the list.")
+                comment: "When a user is in the process of entering an address, a screen pops up prompting the user if they want to use a saved address. This string is used as the title label of the screen.")
```

Comment:
Was this string already exported? If so we might need a new key for the string to be re-exported properly to l10n 👀

Reply:
Hey, no, comments don't matter at all. Only the value does. Well, I guess, the table-name as well, technically. :)

Reply:
thanks for confirming @adudenamedruby! Resolving as no new key is needed if its only a change in comment.